encrypted-media-respec.html

```diff
@@ -3023,17 +3027,17 @@
         <ul>
           <li>
             <p>
-              Access network resources, either local or remote, except via the user agent, or as part of the user agent, as explicitly permitted by this specification.
+              Access network resources, either local or remote, except via the user agent as explicitly permitted by this specification.
```

Comment:
@jdsmith3000 had commented that the CDM being "part of the user agent" was an explicitly allowed implementation choice  (per the paragraph moved back to the CDM definition above). I understood this to be the reason for the "or as part of the user agent" text. However, this case is IMO a subset of "via the user agent" in the sense that a department of FedEx can still send something via FedEx.